man/rgbasm.5

```diff
@@ -355,17 +355,21 @@ The following functions are designed to operate with fixed-point numbers:
 .It Fn DIV x y Ta Fixed-point division
 .It Fn MUL x y Ta Fixed-point multiplication
 .It Fn FMOD x y Ta Fixed-point modulo
-.It Fn POW x y Ta $x$ to the $y$ power
-.It Fn LOG x y Ta Logarithm of $x$ to the base $y$
+.It Fn POW x y Ta $x sup y$
```

Comment:
Double-check how this renders in `man` pages.

Reply:
![image](https://github.com/user-attachments/assets/9e7f5f54-26c7-43cd-8d04-a1a979b4417e)

Reply:
Yeah, <code><em>xy</em></code> looks like multiplication, not exponentiation. :P

Reply:
[Discussion of exponent rendering in an earlier thread](https://github.com/gbdev/rgbds/pull/1455/files#r1706839210)